config/global-constants/src/lib.rs

```diff
@@ -18,6 +18,7 @@ pub const OPERATOR_KEY: &str = "operator";
 pub const OWNER_ACCOUNT: &str = "owner_account";
 pub const OWNER_KEY: &str = "owner";
 pub const VALIDATOR_NETWORK_KEY: &str = "validator_network";
+pub const SHARED_VAL_NETADDR_KEY: &str = "shared_val_netaddr_key";
```

Comment:
oh god that is an ugly undecipherable string :P
what happened to validator_network_address_encryption_key
super long, but very descriptive

we could even add a shared in their...

maybe leveraging both of ours:
shared_validator_network_address_key
(it isn't an encryption key...)

Reply:
That long isn't great either. How about "network_address_key"?

Reply:
i thought we already discussed naming in the previous PR? i feel "validator_network_address_encryption_key" and "shared_validator_network_address_key" are both too long and "network_address_key" is too ambiguous. hence, "shared_val_netaddr_key" : p

Reply:
I'm fine with "shared_val_netaddr_key" BTW.